lib/method/PACMethod_ssh.pm

```diff
@@ -623,10 +620,20 @@ sub _buildGUI
     # Button(s) callback(s)
     $w{btnadd}->signal_connect('clicked', sub {
         my $local_port = 1;
+        my @usedPorts = ();
+
         $$self{cfg} = $self->get_cfg();
-        my $next = () = $$self{cfg} =~ /-[LR]/g;
-        if ($next) {
-            $local_port = $next+1;
+        my @ports = split /-[LR]/,$$self{cfg};
```

Comment:
@ports would include parameters that aren't ports, right? Stuff like -A -X -C from "other options". Isn't that a problem?

Reply:
Asbru is putting them at the beginning so they are discarded.

Reply:
Added your implementation